<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基带传输模拟器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>信源编码</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnSumm">
               <property name="text">
                <string>统计数据</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="btnOpenFile">
               <property name="text">
                <string>打开文件</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,4,2,2,1,7">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="horizontalSpacing">
              <number>10</number>
             </property>
             <item row="0" column="3">
              <widget class="QPushButton" name="btnStartSouce">
               <property name="text">
                <string>开始编码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QPushButton" name="btnJump1">
               <property name="text">
                <string>进入信道编码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboSource">
               <item>
                <property name="text">
                 <string>香农编码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>费诺编码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>哈夫曼编码</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>编码方式：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btnCalSource">
               <property name="text">
                <string>计算编码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QTextBrowser" name="textBrowser1"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>信道编码</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>信源编码方式：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labSourceCode">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="line_code"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,4,2,1,1,3,9">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <item row="0" column="3">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>速度：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboChannel">
               <item>
                <property name="text">
                 <string>单极性非归零码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>双极性非归零码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>单极性归零码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>AMI</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>HDB3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>CMI</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>信道编码方式：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QSpinBox" name="spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QPushButton" name="btnJump2">
               <property name="text">
                <string>进入信源解码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btnChStart">
               <property name="text">
                <string>开始模拟</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QChartView" name="chartView"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>信源解码</string>
          </property>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>321</width>
             <height>531</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_2">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>40</y>
             <width>321</width>
             <height>531</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="textBrowser_3">
           <property name="geometry">
            <rect>
             <x>710</x>
             <y>40</y>
             <width>381</width>
             <height>531</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionclear_all"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actioninformation"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionclear_all">
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actioninformation">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
